developer-docs-site/docs/tutorials/index.md

```diff
@@ -22,6 +24,22 @@ If running macOS, install the following packages in the order specified to take
     ```
 1. **Poetry**: Install Poetry from [https://python-poetry.org/docs/#installation](https://python-poetry.org/docs/#installation).
 
+## Adjust network and funding source
+
+For development purposes, the tutorials here assume you are working in the Aptos devnet network and therefore interacting with the devnet Aptos node and faucet service for creating and funding accounts. As noted in the relevant tutorials, these targets can be configured with the following environment variables:
+  * `APTOS_NODE_URL`
+  * `APTOS_FAUCET_URL`
+
+Here are the options for funding by network:
+  * *Testnet* - Use the faucet button [in-wallet](../guides/install-petra-wallet.md) or use the testnet faucet at: https://aptoslabs.com/testnet-faucet.
+  * *Devnet* - Use the faucet button [in-wallet](../guides/install-petra-wallet.md)
+
+As mentioned at the top, these tutorials are not intended to be followed in mainnet, where is no faucet in-wallet; you must get funds from an exchange.
```

Comment:
the tutorials all expect the faucet url, this feels jarring and unnecessary.

Reply:
Agreed. I was trying to meet your request:
https://github.com/aptos-labs/aptos-core/issues/5178#issuecomment-1287925793

Where you suggested:
"actually I think the bigger issue here is we need to divide the docs into a section on funding accounts, since mainnet, testnet, and devnet are all different. The CLI also is pegged on devnet, which isn't great."

In response to my reporting the CLI didn't immediately recognize the private key of the account I just created:
https://github.com/aptos-labs/aptos-core/issues/5178#issuecomment-1287925793

Frankly, this funding advice feels incomplete and to your point inconsistent. I see references to `aptos account fund-with-faucet --account default` and https://tap.devnet.prod.gcp.aptosdev.com but don't see a clear picture on all options anywhere.

Perhaps we can draft that from scratch. I am still trying to parse your desires from the bug response above. :-)